src/model/chart-model.ts

```diff
@@ -954,6 +1022,31 @@ export class ChartModel<HorzScaleItem> implements IDestroyable, IChartModelBase
 		return this._options.rightPriceScale.visible ? DefaultPriceScaleId.Right : DefaultPriceScaleId.Left;
 	}
 
+	public seriesPaneIndex(series: Series<SeriesType>): number | null {
+		for (let i = 0; i < this._panes.length; i += 1) {
+			if (this._panes[i].series().includes(series)) {
+				return i;
+			}
+		}
+		return null;
+	}
```

Comment:
```suggestion
	private _seriesPaneIndex(series: Series<SeriesType>): number {
		return this._panes.findIndex((pane: Pane) => pane.series().includes(series));
	}
```
Does it need to be public, and do we need to be able to return `null` if not found?
To me it only looks like it is used in `moveSeriesToPane()` and we don't check for null (only that it is different to the new index)